staging/src/k8s.io/sample-apiserver/hack/update-codegen.sh

```diff
@@ -21,6 +21,9 @@ set -o pipefail
 SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 CODEGEN_PKG=${CODEGEN_PKG:-$(cd "${SCRIPT_ROOT}"; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo ../code-generator)}
 
+# go mod vendor does not set scripts as executable
+chmod +x "${CODEGEN_PKG}"/generate-groups.sh "${CODEGEN_PKG}"/generate-internal-groups.sh
```

Comment:
Just realized this isn't really needed for sample-apiserver because it does https://github.com/kubernetes/kubernetes/pull/73447/commits/18b7fda68d8102834ef0fe1b9d17dd40d6c00fec. An alternate way would be use this approach...

Reply:
that seems slightly better to me

Reply:
Updated